pkg/apis/servicecatalog/types.go

```diff
@@ -443,3 +457,19 @@ const (
 const (
 	FinalizerServiceCatalog string = "kubernetes-incubator/service-catalog"
 )
+
+// ParametersFromSource represents the source of a set of Parameters
+type ParametersFromSource struct {
```

Comment:
Am I correct to assume that we have this direction (ParamsFrom->ParamsFromSource->SecretKeyRef) instead of just ParamsFrom->SecretKeyRef so that we can extend ParamsFromSource later with other things, like pointing to an entire Secret?